test/integration/model/update.test.js

```diff
@@ -35,6 +35,53 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         }));
     });
 
+    it('should not check for notNull Violation for undefined values', function() {
+      const ownerId = 2;
+      let accountRowId;
+      return this.Account.create({
+        ownerId,
+        name: Math.random().toString()
+      }).then(account => {
+        accountRowId = account.get('id');
+        const accountVal = {
```

Comment:
values from `accountVal` can be passed directly to `this.Account.update`, no need to keep values as constant.